### ISSUE TITLE
Find by AccessibilityIdentifier

### ISSUE THREAD

**Issue by gmcadosch** [nalexn/clean-architecture-swiftui#77]
There's a find(viewWithId:) function, but as I already set the accessibilityIdentifier property on all views for UI testing, I needed a new find function. I implemented this as a shortcut:

extension InspectableView {
    func find(_ id: String) throws -> InspectableView<ViewType.ClassifiedView> {
        try find { try $0.accessibilityIdentifier() == id }
    }
}

Would be nice to see something like this integrated into the standard package as probably everyone doing UI testing already has those identifiers set.

**Comment by nalexn** [comment]
Hey, feel free to open a PR for that, great idea!

**Comment by gmcadosch** [comment]
Yes, thought the same, but it‘s a bit long, though. But in regard to the other find versions it‘s probably the right name.
